package.json

```diff
@@ -77,7 +78,8 @@
     "shelljs": "^0.8.3",
     "shx": "^0.3.2",
     "size-limit": "^1.3.2",
-    "typescript": "^2.6.0"
+    "ts-jest": "^24.0.0",
```

Comment:
I'm not sure yet, it' been a couples of years since I used Jest and ts-jest.  All of this configuration was brought over from MobX.  I wanted it to match, and then eventually build from there.  I will look into it though once I move on to converting the tests.

Reply:
everytime a start TS + jest without ts-jest, in the end bring it back in. I think it relates to better reloading, or not doing on compile errors or something like that. So I think it is fine for now.